src/fideslang/models.py

```diff
@@ -839,20 +929,49 @@ class PrivacyDeclaration(BaseModel):
     ingress: Optional[List[FidesKey]] = Field(
         description="The resources from which data is received. Any `fides_key`s included in this list reference `DataFlow` entries in the `ingress` array of any `System` resources to which this `PrivacyDeclaration` is applied."
     )
+    features: List[str] = Field(
+        default=[], description="The features of processing personal data."
+    )
+    legal_basis_for_processing: Optional[LegalBasisForProcessingEnum] = Field(
+        description="The legal basis under which personal data is processed for this purpose."
+    )
+    impact_assessment_location: Optional[str] = Field(
+        description="Where the legitimate interest impact assessment is stored"
+    )
+    retention_period: Optional[str] = Field(
+        description="An optional string to describe the time period for which data is retained for this purpose."
+    )
+    processes_special_category_data: bool = Field(
+        default=False,
+        description="This system processes special category data",
+    )
+    special_category_legal_basis: Optional[SpecialCategoryLegalBasisEnum] = Field(
+        description="The legal basis under which the special category data is processed.",
+    )
+    data_shared_with_third_parties: Optional[bool] = Field(
```

Comment:
If we're already defaulting to `False` I don't think the `Optional` is required here?

```suggestion
    data_shared_with_third_parties: bool = Field(
```

Reply:
yep thanks!